tests/core/math/test_vector2.h

```diff
@@ -39,10 +39,14 @@ namespace TestVector2 {
 
 TEST_CASE("[Vector2] Constructor methods") {
 	const Vector2 vector_empty = Vector2();
+	const Vector2 vector_one = Vector2(0);
```

Comment:
```suggestion
	const Vector2 vector_one = Vector2(0.0);
```

Reply:
i'd argue that the point with this constructor is to write less